src/Http/Request/Body/ContentMediationRequestBodyParser.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Request\Body;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+final class ContentMediationRequestBodyParser implements RequestBodyParser
```

Comment:
Just out of curiosity, how should `mediation` be interpreted in this case?

Reply:
Haha it should be "Negotiation". Don't know how that word got in my head instead. https://developer.mozilla.org/en-US/docs/Web/HTTP/Content_negotiation

Reply:
I'll change it!

Reply:
See 5625e05 (I also fixed a merge conflict in 4954700 because I forgot to pull in your changes and you changed the docblock to a property type, so I had to rename it in that as well).